generators/kubernetes/templates/ingress.yml.ejs

```diff
@@ -22,6 +22,7 @@ metadata:
   name: <%= app.baseName.toLowerCase() %>
   namespace: <%= kubernetesNamespace %>
 spec:
+  ingressClassName: <%= ingressTypeNginx || "nginx" %>
```

Comment:
Probably you mean this
```suggestion
  ingressClassName: <%= ingressType || "nginx" %>
```

Reply:
:see_no_evil: it's been so long....thanks for seeing my silly mistakes >,<